### ISSUE TITLE
Nginx Ingress Controller v1.0.0 is incompatible with Mojaloop Helm v13.0.x releases 

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#2448]
**Summary**:

Nginx Ingress Controller v1.0.0 is incompatible with Mojaloop Helm v13.0.x releases.
Ref: https://kubernetes.io/blog/2021/07/26/update-with-ingress-nginx/

This is due to Nginx Ingress Controller v1.x supports stable K8s APIs for v1.22.x.

A workaround is to install **v0.47.0** of the Nginx-Ingress Controller.

If you are using helm, this can be done as follows:

```bash
$ helm install ingress-nginx ingress-nginx/ingress-nginx --version="3.33.0"
```

**Severity**:
medium

**Priority**:
medium

**Expected Behavior**
Mojaloop Helm Charts should support the latest v1.x version of the Nginx Ingress Controller

**Steps to Reproduce**
  1. Deploy Nginx Ingress Controller v1.0.0
  2. Deploy Mojaloop Helm v13.0.1/2 
  3. Ingress's will not be available.

**Specifications**
  - Component (if known): Mojaloop Helm Charts
  - Version: v13.0.1/2
  - Platform: n/a
  - Subsystem:
  - Type of testing: Manual
  - Bug found/raised by: @tdaly61 

**Notes**:

- Severity when opened:
- Priority when opened:

**Comment by tdaly61** [comment]
hey @mdebarros , do we still need this one given we are moving past the 1.20 and 1.21 releases to 1.22 and beyond where the nginx issues are  moot ? I mean apart from documenting the requirements for the older releases (which we may soon not support) then are we ever gonna do anything about this now ?

**Comment by elnyry-sam-k** [comment]
This is now resolved in [v14.1.0](https://github.com/mojaloop/helm/releases/tag/v14.1.0) but might exist in older releases. Can consider it a known issue for older releases.

v14.1.0 is tested with 1.24 of k8s
